cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/cache.go

```diff
@@ -398,42 +402,47 @@ func (c NamespacedResourceWatcherCache) onDelete(unstructuredObj map[string]inte
 	}
 
 	if delete {
-		err = c.redisCli.Del(c.redisCli.Context(), key).Err()
+		keysremoved, err := c.redisCli.Del(c.redisCli.Context(), key).Result()
 		if err != nil {
 			log.Errorf("Failed to delete value for object [%s] from cache due to: %v", key, err)
 			return err
+		} else {
+			// debugging an intermittent failure
```

Comment:
Not sure if you intend to leave this debugging statement, but probably not?

Reply:
I will take care of it in due time, it's on my list. I want it there for a little while I am actively working on it